components/dashboard/src/components/Modal.tsx

```diff
@@ -93,16 +94,9 @@ export default function Modal(props: {
                     )}
                     {props.title ? (
                         <>
-                            <h3 className="pb-2">{props.title}</h3>
-                            <div
-                                className={
-                                    "border-gray-200 dark:border-gray-800 -mx-6 px-6 " +
-                                    (props.hideDivider ? "" : "border-t border-b mt-2 py-4")
-                                }
-                            >
-                                {props.children}
-                            </div>
-                            <div className="flex justify-end mt-6 space-x-2">{props.buttons}</div>
+                            <ModalHeader>{props.title}</ModalHeader>
+                            <ModalBody hideDivider={props.hideDivider}>{props.children}</ModalBody>
```

Comment:
**fyi**: Not sure where to add this comment, but one modal I'd love to improve after thes changes is the feedback modal. Cross-posting follow up issue (https://github.com/gitpod-io/gitpod/issues/10307) for visibility.